### PR TITLE
Updates for svg download

### PR DESCRIPTION
**Pull request by Jephuff**
- [x] add tooltip for png download in IE

### Distribution of Most Frequently Mutated Genes:
- [x]  add Title: Distribution of Most Frequently Mutated Genes
- [x]  make the y axis label font smaller
- [x]  put a little padding at the top of the SVG/PNG file so it is not flushed at the top

### Overall Survival Plot:
- [x]  add Title: Overall Survival Plot
- [x]  make sure to include the P-value on the download
- [x]  put a little padding at the top of the SVG/PNG file so it is not flushed at the top

### OncoGrid:
- [x]  add Title: Top 50 Mutated Genes by High Impact Mutations
- [x]  add grid legend to download
- [x]  put a little padding at the top of the SVG/PNG file so it is not flushed at the top
- [x] add track legend to download

### Distribution of Most Frequent Mutations:
- [x]  add Title: Distribution of Most Frequent Mutations
- [x]  make the y axis label font smaller
- [x]  make sure to use the shortened Mutation ID on SVG and PNG export
- [x]  put a little padding at the top of the SVG/PNG file so it is not flushed at the top

### Cancer Distribution:
- [x]  add Title: Cancer Distribution
- [x]  make the y axis label font smaller
- [x]  put a little padding at the top of the SVG/PNG file so it is not flushed at the top

### Protein Viewer:
- [x]  add Title: Gene: [Gene Symbol], Transcript: [Transcript ID ([Transcript size] aa). E.g Gene: NFE, Transcript: ENST000399383 (569 aa)
- [x]  put a little padding at the top of the SVG/PNG file so it is not flushed at the top
- [x]  add legend to download

### CODE REVIEW

**Comment by shanewilson** [comment]
this feels important - is it ready to merge?

**Comment by Jephuff** [comment]
It's close, I just need to add a tooltip for IE saying it's currently unsupported and then test

**Comment by shanewilson** [comment]
ready to merge?

**Comment by Jephuff** [comment]
Yep, just pending review.

**Comment by shanewilson** [comment]
who can review this?

@cy @alex-wilmer @cheapsteak any one else work on this code and feel comfortable reviewing this?
otherwise I'm probably just going to accept it to get it in for UAT testing

File: modules/node_modules/@ncigdc/components/SurvivalPlotWrapper.js [hunk 1]
```diff
@@ -107,24 +130,31 @@ const SurvivalPlotWrapper = ({
...
-                  >{l.value}</div>
-                ))
-              }
-            </Row>
+            <div style={graphTitle()}>{TITLE}</div>
+            <div>
+              <Row style={{ justifyContent: 'center', flexWrap: 'wrap', marginTop: '0.5rem' }}>
+                {legend &&
+                  legend.map((l, i) => (
+                    <div className={`legend-${i}`}>
```
Comment (alex-wilmer) [inline modules/node_modules/@ncigdc/components/SurvivalPlotWrapper.js]: key in the wrong spot

**Comment by alex-wilmer** [comment]
other than that, lgtm 👍

File: modules/node_modules/@ncigdc/components/SurvivalPlotWrapper.js (hunk 1 above)
Comment (Jephuff) [inline modules/node_modules/@ncigdc/components/SurvivalPlotWrapper.js]: oops! fixed.

**Pull request merged**